flight/PiOS/ChibiOS/os/common/startup/ARMCMx/compilers/GCC/mk/startup_stm32f4xx.mk

```diff
@@ -6,7 +6,9 @@ STARTUPASM = $(CHIBIOS)/os/common/startup/ARMCMx/compilers/GCC/crt0_v7m.S
 
 STARTUPINC = $(CHIBIOS)/os/common/startup/ARMCMx/compilers/GCC \
              $(CHIBIOS)/os/common/startup/ARMCMx/devices/STM32F4xx \
-             $(CHIBIOS)/os/common/ext/CMSIS/include \
-             $(CHIBIOS)/os/common/ext/CMSIS/ST/STM32F4xx
+             $(PIOS_DEVLIB)/Libraries/CMSIS/Device/ST/STM32F4xx \
+             $(PIOSCOMMON)/Libraries/CMSIS/Include
+#             $(CHIBIOS)/os/common/ext/CMSIS/include \
+#             $(CHIBIOS)/os/common/ext/CMSIS/ST/STM32F4xx
```

Comment:
This is where you strike problems with upgrading ChibiOS. Their CMSIS impl. doesn't seem compatible with StdPeriph etc. and vice-versa.